content/events/2024-in-person-conference/_index.md

```diff
@@ -5,50 +5,82 @@ description = "Nordic RSE – Research Software Engineers in the Nordics – Con
+## Where
+
+On the campus of Aalto University in Otaniemi, Espoo, Finland.
+
+
+### How you can help
+
+ - Submit a contribution! The conference will be more interesting with your contribution on the agenda.
+ - Encourage your colleagues to submit a contribution. Since the term RSE is still fairly new, they might not know about it yet and be hesitant to submit something. 
+ - [Share the announcement with others](/events/2024-in-person-conference/). 
+
+
 ## Call for Contributions
 
-We will soon open a call for contributions.
- 
+The call for contributions is open!
 
-You can also help us by [sharing the announcement with others](/events/2023-online-unconference/share/). 
+<a class="btn btn-primary btn-lg" href="https://indico.neic.no/event/260/abstracts/" target="_blank" rel="noreferrer noopener" role="button">Submit a Contribution</a>
 
-And of course we will all follow a [Code of
-Conduct](https://nordic-rse.org/about/code-of-conduct/).
+We invite you to submit contributions on any topics related to research software. Whether you are using it or developing it yourself,
+we are interested in your experience and expertise. Topics are divided roughly into 3 sessions, "Best Practices", "Technical" and
+"Soft Skills". But don't let this limit you, many topics are welcome. Just submit your idea and we can discuss and find a spot for it!
 
-All formats of communication are welcome (or check what we have done [last year](/events/2023-online-unconference/)):
-- *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
+All formats of communication are welcome (or check what we have done [in the unconference](/events/2023-online-unconference/)):
+- *Discussion topics*: is software, research software community or research software engineering something close to your heart? Come chat with other enthusiasts!
 - *Demonstrations:* show us some tools or software that you like
-- *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (max 3 hours in a day)
+- *Workshops or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (max 3 hours in a day)
 - *Talks*: teach us something new
-- *On-going projects*: share with others something unfinished and almost working that you would like to get feedback on
-- *Something else*: is there something you would like to share with others that
+- *On-going projects*: share something unfinished or almost working that you would like to get feedback on
+- *Something else*: is there something you would like to share that
   does not fit any of the previous points? We are looking forward to hearing
   about it!
 
+The open session will include on day contributions, topics that arise during the conference.
+
+And of course we will all follow a [Code of Conduct](https://nordic-rse.org/about/code-of-conduct/).
+
+### Price
+
+The participation fee will be 50€-100€ and will cover a conference dinner on Thursday.
```

Comment:
"and will _also_ cover"

Otherwise it sounds like that's the only thing it will be covered, which for 50-100€ it's a lot. Better yet if we name two or three things that will be covered.

Reply:
Currently reads "will also cover lunch and refreshments". Could add venue, but that seems obvious.

Reply:
Updated following the last comment and ended up removing this part.